freqtrade/freqtradebot.py

```diff
@@ -1181,7 +1181,7 @@ def _notify_sell(self, trade: Trade, order_type: str, fill: bool = False) -> Non
             'current_rate': current_rate,
             'profit_amount': profit_trade,
             'profit_ratio': profit_ratio,
-            'sell_reason': trade.sell_reason,
+            'sell_reason': trade.sell_reason  + (f' ({trade.buy_tag})' if trade.buy_tag else ''),
```

Comment:
i'd expose "buy_tag" in this call seperately (below too, obviously).

By combining the data here, you're forcing consumers to use it this way (this might be especially problematic for webhooks, where users can configure the provided data themselfs).

in my opinion, this "combination" should be happening in the display layer (which would be the rpc/* files) - where necessary obviously (most likely only in telegram).